CHANGELOG.md

```diff
@@ -1,8 +1,146 @@
+  rec.spawn().exit_on_failure();
+
+  for (int step = 0; step < 64; ++step) {
+      rec.set_time_sequence("step", step);
+      rec.log("scalar", rerun::Scalar(std::sin(static_cast<double>(step) / 10.0)));
+  }
+  ```
+
+  Using `send()` (fast, memory efficient):
+  ```c++
+  const auto rec = rerun::RecordingStream("rerun_example_send_columns");
+  rec.spawn().exit_on_failure();
+
+  std::vector<double> scalar_data(64);
+  for (size_t i = 0; i < 64; ++i) {
+      scalar_data[i] = sin(static_cast<double>(i) / 10.0);
+  }
+  std::vector<int64_t> times(64);
+  std::iota(times.begin(), times.end(), 0);
+
+  auto time_column = rerun::TimeColumn::from_sequence_points("step", std::move(times));
+  auto scalar_data_collection =
+      rerun::Collection<rerun::components::Scalar>(std::move(scalar_data));
+
+  rec.send_columns("scalars", time_column, scalar_data_collection);
+  ```
+</details>
+<details>
+  <summary>Rust timeseries</summary>
+
+  Using `log()` (slow, memory inefficient):
+  ```rust
+  let rec = rerun::RecordingStreamBuilder::new("rerun_example_scalar").spawn()?;
+
+  for step in 0..64 {
+      rec.set_time_sequence("step", step);
+      rec.log("scalar", &rerun::Scalar::new((step as f64 / 10.0).sin()))?;
+  }
+  ```
+
+  Using `send()` (fast, memory efficient):
+  ```rust
+  let rec = rerun::RecordingStreamBuilder::new("rerun_example_send_columns").spawn()?;
+
+  let timeline_values = (0..64).collect::<Vec<_>>();
+  let scalar_data: Vec<f64> = timeline_values
+      .iter()
+      .map(|step| (*step as f64 / 10.0).sin())
+      .collect();
+
+  let timeline_values = TimeColumn::new_sequence("step", timeline_values);
+  let scalar_data: Vec<Scalar> = scalar_data.into_iter().map(Into::into).collect();
+
+  rec.send_columns("scalars", [timeline_values], [&scalar_data as _])?;
+  ```
+</details>
+
+#### Background compaction
+
+The Rerun datastore now continuously compacts data as it comes in, in order find a sweet spot between ingestion speed, query performance and memory overhead.
+
+This is very similar to, and has many parallels with, the [micro-batching mechanism running on the SDK side](https://rerun.io/docs/reference/sdk-micro-batching).
+
+You can read more about this in the [dedicated documentation entry](https://rerun.io/docs/reference/store-compaction?speculative-link).
+
+#### Post-processing of RRD files
+
+To help improve efficiency for completed recordings, Rerun 0.18 introduces some new commands for working with rrd files.
+
+Multiple files can be merged, whole entity paths can be dropped, and chunks can be compacted.
+
+You can read more about it [in the new CLI reference manual](https://rerun.io/docs/reference/cli), but to give a sense of how it works the below example merges all recordings in a folder and runs chunk compaction using the `max-rows` and `max-bytes` settings:
+```sh
+rerun rrd compact --max-rows 4096 --max-bytes=1048576 /my/recordings/*.rrd > output.rrd
+```
+
+### ⚠️  Breaking changes
```

Comment:
Should we include the regression on "plot hole" somewhere here, or in another "Known issues" section?
- https://github.com/rerun-io/rerun/pull/6586

Reply:
Oh nice catch, I thought we shipped that in 0.17.

I'll add this to the breaking change section... unless I fix it today?!

Reply:
https://github.com/rerun-io/rerun/pull/7184